### PR TITLE
login creation

### PR DESCRIPTION
**Pull request by Cdogonza**
Just creating the login screen layout, not the logic yet

### CODE REVIEW

**Comment by danacastroman** [comment]
You have to pull last `dev` changes and do a rebase, use `git rebase dev` on your branch to merge dev into your branch.
After that fix conflicts and put your login component inside the `pages` folder

File: angular.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
       "projectType": "application",
       "schematics": {
         "@schematics/angular:component": {
-          "style": "scss"
+          "style": "scss",
+          "skipTests": true
```
Comment (alfonsoLeandro) [inline angular.json]: Good job adding the skipTests rule, so that we can avoid deleting the tests files from now on, as we are not using tests

**Comment by alfonsoLeandro** [comment]
LGMT, just go over Dana's comments and we are ready to merge

**Comment by YOGURSIO** [comment]
Hey, check your .gitignore, something is clearly not right with it.

File: client/src/app/pages/login/login.component.scss
Comment (alfonsoLeandro) [inline client/src/app/pages/login/login.component.scss:1]: This file uses a lot of px as a unit of measure, please try using relative units more frequently

File: package-lock.json
Comment (alfonsoLeandro) [inline package-lock.json:1]: delete this file, as it is outside both the client and the server project

File: package.json
Comment (alfonsoLeandro) [inline package.json:1]: delete this file, as it is outside both the client and the server project

File: client/src/app/pages/login/login.component.html [hunk 2]
```diff
@@ -1 +1,16 @@
...
+        <img src="../../../assets/images/patitas.png" alt="">
+
+</div>
+<div class="form">
+    <input type="text" placeholder="Email">
+    <input type="password" placeholder="Password">
+    <i class="fa-sharp fa-solid fa-eye-slash"></i>
+    <p>¿Olvidaste tu Contraseña? <a href="#">Recuperar</a></p>
+    <button>Iniciar Sesion</button>
+    <p [routerLink]="">¿No tienes una cuenta? <a href="#">Registrate</a></p>
```
Comment (danacastroman) [inline client/src/app/pages/login/login.component.html]: I recommend you to change this line to `<p>¿No tienes una cuenta? <a routerLink="/register">Registrate</a></p>` to respect the SPA concept.

File: client/src/app/service.service.ts
Comment (danacastroman) [inline client/src/app/service.service.ts:1]: When is this file used? If you don't need it, delete it or put it in the `app/services` folder.

Comment (Cdogonza) [reply client/src/app/service.service.ts:1]: I will place it in the services folder and put a more appropriate name, thanks a lot

File: client/src/app/pages/login/login.component.html (hunk 2 above)
Comment (Cdogonza) [reply client/src/app/pages/login/login.component.html]: Im agree! i will change

File: client/src/app/pages/login/login.component.scss
Comment (alfonsoLeandro) [reply client/src/app/pages/login/login.component.scss:1]: Well done changing the units! Does it still look good according to the figma file?

File: client/src/app/pages/login/login.component.scss [hunk 3]
```diff
@@ -32,38 +24,32 @@ margin: auto;
...
-        width: 125px;
-        height: 40px;
+        gap: 5em;
+        width: 10em;
+        height: 3em;
         background: #2CBBC3;
         border-radius: 100px;
         border-color: #2CBBC3;
         border-style: none;
         font-family: 'Roboto';
```
Comment (alfonsoLeandro) [inline client/src/app/pages/login/login.component.scss]: Remember to remove all the "font-family" and "font-style" attributes from this file, as they override a rule with the same value

File: client/src/app/services/post-login.service.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { Injectable } from '@angular/core';
 @Injectable({
   providedIn: 'root'
 })
-export class ServiceService {
+export class PostLoginService {
```
Comment (alfonsoLeandro) [inline client/src/app/services/post-login.service.ts]: What do you mean by "PostLogin"?

File: client/src/app/pages/login/login.component.scss (hunk 3 above)
Comment (Cdogonza) [reply client/src/app/pages/login/login.component.scss]: done

**Comment by alfonsoLeandro** [comment]
PR discarded - we need to move on. The new login page in the new ionic frontend has already been made, so no need for this PR to be merged, as the angular frontend will be discarded eventually too.

**Pull request closed without merging**